packages/pie-toolbox/src/code/mask-markup/components/blank.jsx

```diff
@@ -29,7 +29,6 @@ const useStyles = withStyles(() => ({
     height: 'auto',
     maxWidth: '374px',
     position: 'relative',
-    borderRadius: '3px'
```

Comment:
I think this was deleted by mistake. It was added here: https://github.com/pie-framework/pie-lib/pull/1343/files#diff-3d58c970e7e794786201fb08669c19efa1fe46f20c9f088b4443bbc96967eef0R32 
as it was a change requested in: https://illuminate.atlassian.net/browse/PD-3950